onnxruntime/test/providers/cpu/math/element_wise_ops_test.cc

```diff
@@ -252,7 +252,7 @@ TEST(MathOpTest, Div_int64) {
   test.AddInput<int64_t>("A", {3}, {4, 8, 8});
   test.AddInput<int64_t>("B", {3}, {2, 3, 4});
   test.AddOutput<int64_t>("C", {3}, {2, 2, 2});
-  test.Run();
+  test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kTensorrtExecutionProvider});
```

Comment:
for types that tensorrt doesn't support, doesn't we fall back to cpu provider? does this test still fail because of GetCapability type checking?

Reply:
Yes. That's the fundamental issue of the parser. Parser claims TRT can run the node but it failed while parsing the node because of various of reasons (either parser itself's issue or discrepancy between parser and trt's capability)